### ISSUE TITLE
Python Error Running Script

### ISSUE THREAD

**Issue by SoccerNerd** [cbanack/comic-vine-scraper#490]
Installed on new PC and I am getting the following Python error. This did not occur on old system. 

I do have Python 3.11 installed on my new PC, not sure if this would result in issues. 

------------------- PYTHON ERROR ------------------------
Caught StandardError: Exception has been thrown by the target of an invocation.
Traceback (most recent call last):
  File "C:\Users\joe.zucco\AppData\Roaming\cYo\ComicRack\Scripts\Comic Vine Scraper\scrapeengine.py", line 142, in scrape
  File "C:\Users\joe.zucco\AppData\Roaming\cYo\ComicRack\Scripts\Comic Vine Scraper\scrapeengine.py", line 209, in _ScrapeEngine__scrape
  File "C:\Users\joe.zucco\AppData\Roaming\cYo\ComicRack\Scripts\Comic Vine Scraper\scrapeengine.py", line 628, in _ScrapeEngine__sort_books
  File "C:\Users\joe.zucco\AppData\Roaming\cYo\ComicRack\Scripts\Comic Vine Scraper\scrapeengine.py", line 596, in _ScrapeEngine__compare_books
  File "C:\Users\joe.zucco\AppData\Roaming\cYo\ComicRack\Scripts\Comic Vine Scraper\comicbook.py", line 80, in <lambda$87>
  File "C:\Users\joe.zucco\AppData\Roaming\cYo\ComicRack\Scripts\Comic Vine Scraper\comicbook.py", line 239, in _ComicBook__unique_series_s

**Comment by cbanack** [comment]
Well, the scraper plugin uses Iron Python 2.7, but that is installed with the rest of ComicRack, so I don't see how having Python 3 on your system would affect it.

I believe you can press Ctrl-shift-L  while a scraper window has keyboard focus and it should save out a more complete log file that might give you a better hint about what's going on.

**Comment by SoccerNerd** [comment]
I tried Ctrl-shift-L and did not generate any additional information in the log file. I am also getting some other ComicRack minor errors so might be a permissions issue on the new PC. Will play around with it some more.

**Comment by SoccerNerd** [comment]
OK, this might have been related to another issue with ComicRack. I was getting an annoying pop up everytime I started that was complaining about "This implementation is not part of the Windows Platform FIPS validated cryptographic algorithms". But the app seemed to work fine with the exception of Comicvine. 

I found a solution to the ComicRack error by setting the HKEY_LOCAL_MACHINE\System\CurrentControlSet\Control\Lsa\FipsAlgorithmPolicy key in the registry to 0 (off). 

I no longer received the ComicRack pop-up warning, and it seems to have also allowed Comicvine to run correctly as well. 

Posting here in case anyone else runs into a similar issue.
